### PR TITLE
Recheck unification in TWC, CWC, MWC

### PR DESCRIPTION
**Pull request by fajar13k**
- Leftover map links
- Some missing players
- Reformat tables
- Minor naming change
- Sort alphabetically participants and staffs
- Replace pictures to `.jpg` except it has alpha channels
- Remark errors are from missing beatmaps
- Ready for review

### CODE REVIEW

File: wiki/Tournaments/TWC/3/en.md [hunk 1]
```diff
@@ -380,31 +374,31 @@ This competition has come to an end and resulted in the following podium:
...
+[flag_NO]: /wiki/shared/flag/NO.gif "Norway"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
+[flag_SV]: /wiki/shared/flag/SV.gif "El Salvador"
+[flag_TH]: /wiki/shared/flag/TH.gif "Thailand"
+[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
+[flag_UA]: /wiki/shared/flag/UA.gif "Ukraine"
+[flag_US]: /wiki/shared/flag/US.gif "United States"
+[flag_VE]: /wiki/shared/flag/VE.gif "Venezuela"
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/3/en.md:404]: unify Group Stage headers in mappool section (`Group Stage (Week X)` -> `Group Stage week X`)

File: wiki/Tournaments/CWC/2016/en.md [hunk 2]
```diff
@@ -175,7 +169,7 @@ The osu!catch World Cup 2016 was run by various community members by distributin
 - NoMod
   - [Seiryu - Ultramarine (ZiRoX) \[Spec's Abyss\]](https://osu.ppy.sh/beatmapsets/275991#fruits/635701)
```
Comment (TPGPL) [inline wiki/Tournaments/CWC/2016/en.md:170]: L151 escape strikethrough

**Review by TPGPL — CHANGES_REQUESTED** [review]
maybe if the beatmap is unaccessable even through bloodcat, then maybe remove the link completely?

**Comment by fajar13k** [comment]
Sure we can remove them, but I may include its beatmaps id in the comment
e.g.: `<!-- map is missing (beatmaps id) -->`

**Review by TPGPL — APPROVED** [review]

**Pull request merged**